src/lib/moment/locale.js

```diff
@@ -18,16 +18,13 @@ export function locale (key) {
     }
 }
 
-export var lang = deprecate(
-    'moment().lang() is deprecated. Instead, use moment().localeData() to get the language configuration. Use moment().locale() to change languages.',
```

Comment:
This deprecation wrapper got moved to the prototype definition, the same way we handle all the other deprecated APIs.